src/lexer.l

```diff
@@ -113,119 +113,126 @@ PSL_CONT        ^{SPACE}*({PSL_COMMENT}|"--")
+OUT                  ?i:out
+BUFFER               ?i:buffer
+BUS                  ?i:bus
+REGISTER             ?i:register
+UNAFFECTED           ?i:unaffected
+SIGNAL               ?i:signal
+PROCESS              ?i:process
+WAIT                 ?i:wait
+REPORT               ?i:report
+INOUT                ?i:inout
+LINKAGE              ?i:linkage
+VARIABLE             ?i:variable
+FOR                  ?i:for
+TYPE                 ?i:type
+RANGE                ?i:range
+TO                   ?i:to
+DOWNTO               ?i:downto
+SUBTYPE              ?i:subtype
+UNITS                ?i:units
+PACKAGE              ?i:package
+LIBRARY              ?i:library
+USE                  ?i:use
+NULL                 ?i:null
+FUNCTION             ?i:function
+IMPURE               ?i:impure
+PURE                 ?i:pure
+RETURN               ?i:return
+ARRAY                ?i:array
+OTHERS               ?i:others
+ASSERT               ?i:assert
+SEVERITY             ?i:severity
+ON                   ?i:on
+MAP                  ?i:map
+IF                   ?i:if
+THEN                 ?i:then
+ELSE                 ?i:else
+ELSIF                ?i:elsif
+BODY                 ?i:body
+WHILE                ?i:while
+LOOP                 ?i:loop
+AFTER                ?i:after
+ALIAS                ?i:alias
+MOD                  ?i:mod
+ATTRIBUTE            ?i:attribute
+PROCEDURE            ?i:procedure
+EXIT                 ?i:exit
+REM                  ?i:rem
+WHEN                 ?i:when
+CASE                 ?i:case
+TRANSPORT            ?i:transport
+INERTIAL             ?i:inertial
+REJECT               ?i:reject
+BLOCK                ?i:block
+WITH                 ?i:with
+SELECT               ?i:select
+GENERATE             ?i:generate
+ACCESS               ?i:access
+POSTPONED            ?i:postponed
+FILE                 ?i:file
+OPEN                 ?i:open
+UNTIL                ?i:until
+RECORD               ?i:record
+NEW                  ?i:new
+SHARED               ?i:shared
+NEXT                 ?i:next
+SLL                  ?i:sll
+SRL                  ?i:srl
+SLA                  ?i:sla
+SRA                  ?i:sra
+ROL                  ?i:rol
+ROR                  ?i:ror
+GROUP                ?i:group
+LABEL                ?i:label
+LITERAL              ?i:literal
+GUARDED              ?i:guarded
+REVRANGE             ?i:reverse_range
+PROTECTED            ?i:protected
+CONTEXT              ?i:context
+ERROR                ?i:error
+WARNING              ?i:warning
+DISCONNECT           ?i:disconnect
+FORCE                ?i:force
+RELEASE              ?i:release
+PARAMETER            ?i:parameter
+ALWAYS               ?i:always
+DEFAULT              ?i:default
+CLOCK                ?i:clock
+NEVER                ?i:never
+EVENTUALLY           ?i:eventually
+NEXT_A               ?i:next_a
+NEXT_E               ?i:next_e
+NEXT_EVENT           ?i:next_event
+ASSUME               ?i:assume
+ASSUME_GUARANTEE     ?i:assume_guarantee
```

Comment:
Let's just use `ASSUMEG` and `RESTRICTG` here too then we don't need to change the indentation of everything.